storage-operator/pkg/salt/client.go

```diff
@@ -195,12 +195,17 @@ func (self *Client) GetVolumeSize(
 			err, "disk.dump failed (target=%s, path=%s)", nodeName, devicePath,
```

Comment:
Commit message: 

```diff
- reviewers comments I now use the correct path for the volume
+ reviewers' comments I now use the correct path for the volume
```

```diff
- failing (because the path doesn't exists).
+ failing (because the path doesn't exist).
```

Reply:
Done